tests/test_digest_config.py

```
@@ -0,0 +1,159 @@
+    (tmp_path / ".ovp" / "digest.yaml").write_text(
+        yaml.safe_dump({"intake_event_types": []}), encoding="utf-8"
+    )
+    cfg = load_digest_config(tmp_path)
+    assert "article_processed" in cfg.intake_event_types
+
+
+def test_config_is_immutable():
+    """Frozen dataclass + tuple sequence — no field assignment, no
+    list mutation possible."""
+    cfg = DigestConfig()
+    with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)